pwiz_tools/Skyline/TestUtil/TestFunctional.cs

```diff
@@ -2603,72 +2602,78 @@ public static IList<DbRefSpectra> GetRefSpectra(SQLiteConnection connection)
         {
             var list = new List<DbRefSpectra>();
             var hasAnnotations = SqliteOperations.TableExists(connection, @"RefSpectraPeakAnnotations");
-            using (var select = new SQLiteCommand(connection) { CommandText = "SELECT * FROM RefSpectra" })
-            using (var reader = @select.ExecuteReader())
-            {
-                var iAdduct = reader.GetOrdinal("precursorAdduct");
-                var iIonMobility = reader.GetOrdinal("ionMobility");
-                var iIonMobilityHighEnergyOffset = reader.GetOrdinal("ionMobilityHighEnergyOffset");
-                var iCCS = reader.GetOrdinal("collisionalCrossSectionSqA");
-                var noMoleculeDetails = reader.GetOrdinal("moleculeName") < 0; // Also a cue for presence of chemicalFormula, inchiKey, and otherKeys
-                while (reader.Read())
+            using (var select = new SQLiteCommand(connection))
+            {
+                select.CommandText = "SELECT * FROM RefSpectra";
+                using (var reader = select.ExecuteReader())
```

Comment:
It might be good to have a using declaration here instead of a using statement with curly braces, just to reduce the amount that the code has to be indented.

Reply:
Can do, though I was trying to minimize code thrash. Do you feel it's important?

Reply:
Not important, but if you were to replace one using statement with a using declaration things would end up being indented the same amount as they were before.

Reply:
Ah, I see. Yes, actually less code thrash that way. Thanks!